src/universal/modules/email/components/SummaryEmail/ExportToCSV.tsx

```
@@ -46,6 +47,12 @@ interface CSVRow {
 }
 
 class ExportToCSV extends Component<Props> {
+  componentDidMount () {
+    if (this.props.urlAction === 'csv') {
+      this.exportToCSV().catch()
```

Why the empty catch clause?